### PR TITLE
Rename assignment model

### PR DESCRIPTION
**Pull request by nbuonin**

### CODE REVIEW

**Comment by zmustapha** [comment]
Travis testing has not kicked in automatically?

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,20 @@
 language: python
 dist: bionic 
-sudo: false
+sudo: true
 addons:
+  postgresql: "10"
```
Comment (sdreher) [inline .travis.yml:5]: Not sure this is needed.

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
   apt:
     packages:
       - libgconf-2-4
+      - postgresql-10
```
Comment (sdreher) [inline .travis.yml:9]: Or these two packages.

File: .travis.yml [hunk 3]
```diff
@@ -1,10 +1,20 @@
...
+      - postgresql-client-10
```
Comment (sdreher) [inline .travis.yml:10]: Not sure these two packages are needed.

Comment (nbuonin) [reply .travis.yml]: I pulled these from the conf

**Pull request merged**